plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -2370,142 +2338,6 @@ bool producer_plugin_impl::process_unapplied_trxs(const fc::time_point& deadline
    return !exhausted;
 }
 
-void producer_plugin_impl::process_scheduled_and_incoming_trxs(const fc::time_point& deadline, unapplied_transaction_queue::iterator& itr) {
-   // scheduled transactions
-   int    num_applied         = 0;
-   int    num_failed          = 0;
-   int    num_processed       = 0;
-   bool   exhausted           = false;
-   double incoming_trx_weight = 0.0;
-
-   auto&              blacklist_by_id     = _blacklisted_transactions.get<by_id>();
```

Comment:
I think we should be able to also remove `_blacklisted_transactions` (and `remove_expired_blacklisted_trxs`) now.